pkgs/development/python-modules/tsfresh/default.nix

```diff
@@ -0,0 +1,90 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, requests
+, numpy
+, pandas
+, scipy
+, statsmodels
+, patsy
+, scikit-learn
+, tqdm
+, dask
+, distributed
+, stumpy
+, cloudpickle
+, pytestCheckHook
+, mock
+, matplotlib
+, seaborn
+, ipython
+, notebook
+, pandas-datareader
+}:
+
+buildPythonPackage rec {
+  pname = "tsfresh";
+  version = "0.20.1";
+  format = "setuptools";
+
+  disable = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "blue-yonder";
+    repo = "tsfresh";
+    rev = "v${version}";
+    hash = "sha256-JmdP/6aTnuYsBRiRq9zZng3xNYhOdr9V8bp1trAv508=";
+  };
+
+  patches = [
+    # The pyscaffold is not a buil dependency but just a python project bootstrapping tool, so we do not need it
+    ./remove-pyscaffold.patch
```

Comment:
Can we use `substituteInPlace` instead?
The patch file makes updating a bit messy.

Maybe we could just make this line a blank line.
https://github.com/blue-yonder/tsfresh/blob/v0.20.1/setup.cfg#L34

Reply:
Should be better now, just removing one lin ;)